### DISCUSSION TITLE
Woocommerce Integration

### DISCUSSION THREAD

**Discussion by mmojadad** [chatwoot/chatwoot#5741] [General]
I am interest to know how is it possible integrate woocommerce/wordpress with chatwoot, For example if someone connect with agent, the agent can see the customers orders history,Customer Shop Details,Abandoned Cart Recovery etc from the Chatwoot dashboard

[a comment by an automated account is not shown]

**Comment by mTondenier** [comment]
Hello,

Still not possible ? I see a coming soon integration on the website

**Comment by mctlisboa** [comment]
If you're looking for a simple integration between Chatwoot and WooCommerce/WordPress, you might want to check out my plugin: [Chatwoot Identity Validation for WooCommerce](https://github.com/mctlisboa/chatwoot-identity-validation).

It handles many of the requirements you mentioned:

- Shows customer order history (total orders and total spent)
- Shares customer registration date and location
- Securely identifies customers using HMAC validation
- Passes WooCommerce customer data to your Chatwoot agents

The plugin adds this info directly to the customer profile in the Chatwoot dashboard, so your agents can see relevant customer data without leaving the chat interface.

It's straightforward to set up - just install the plugin, enter your Chatwoot credentials, and the integration starts working immediately. There's also a settings page to customize how the widget appears on your site.


Let me know if you have any questions about using it!

**Reply by mmojadad** [reply]
Hi!
That’s great news!
I will definitely implement and test it, and I will keep you updated on the results.
Also, I am with you on this project to further develop and add more complementary ideas in the future.
I will be traveling over the next two weeks, so I won’t be able to review or install it practically. However, I will start immediately after my trip.
